apps/src/music/views/ProjectBeats.tsx

```diff
@@ -19,11 +20,23 @@ const ProjectBeats: React.FunctionComponent<{channelId: string}> = ({
       <Lab2Wrapper>
         <MetricsAdapter />
         <ProjectContainer channelId={channelId}>
-          <MusicView inIncubator={true} />
+          <DeferredMusicView />
         </ProjectContainer>
       </Lab2Wrapper>
     </Provider>
   );
 };
 
 export default ProjectBeats;
+
+// Defers loading MusicView until the channel has been loaded. This ensures
+// that all project data has been loaded before mounting MusicView.
+const DeferredMusicView: React.FunctionComponent = () => {
```

Comment:
Small update here: I added this so that the Project Beats page would function more like other Lab2 pages, so we can make the same assumptions about project data being loaded before the view is mounted.